Toolbox/scripts/util_info.sh

```diff
@@ -4,7 +4,11 @@
 
 #Firmware/unit info:
 export VERSION="$(cat /net/rcc/dev/shmem/version.txt | grep "Current train" | sed 's/Current train = //g' | sed -e 's|["'\'']||g' | sed 's/\r//')"
-export FAZIT=$(cat /tmp/fazit-id);
+if [ -e /tmp/fazit-id ]; then
```

Comment:
FAZIT could also be found in persistance. Maybe this is more failsafe?

`FAZIT="$(on -f mmx /eso/bin/dumb_persistence_reader 0 3221291024 2> /dev/null | $XXD -r -p 2> /dev/null)"`

Reply:
XXD doesn't exist on the MIB though, not in any of the firmware I've search through at least. I have actually compiled a copy of it, but haven't tested it yet.